manifests/mirror.pp

```diff
@@ -85,22 +78,28 @@
 
     $exec_aptly_mirror_create_require = [
       Package['aptly'],
-      File['/etc/aptly.conf'],
       Exec["aptly_mirror_gpg-${title}"],
     ]
   } else {
     $exec_aptly_mirror_create_require = [
       Package['aptly'],
-      File['/etc/aptly.conf'],
     ]
   }
 
-  $cmd_options_string = join(reject(join_keys_to_values(merge($default_cmd_options, $cmd_options), '='), '.*=$'), ' ')
-  $cmd_string         = rstrip("${aptly_cmd} create ${cmd_options_string} ${title} ${location} ${release} ${components}")
+  $cli_options_string = join(reject(join_keys_to_values($cli_options, '='), '.*=$'), ' ')
```

Comment:
Yeah, which by the way is not necessary anymore since I removed the defaults. I'll get rid of it to make it simpler.